### PR TITLE
708 update change password according to new design

### PR DESCRIPTION
**Pull request by ayushicopods**

### CODE REVIEW

File: app/routes/$workspaceId.tsx [hunk 1]
```diff
@@ -38,7 +53,66 @@ export const loader: LoaderFunction = async ({ request, params }) => {
   return json({ workspaces, currentWorkspaceId })
 }
 
-export const action: ActionFunction = async ({ request }) => {
+export const action: ActionFunction = async ({ request, params }) => {
+  const formData = await request.formData() // getting formData
+  // action will perform if match with specific formData
```
Comment (yash6102000) [inline app/routes/$workspaceId.tsx:58]: we are not performing any action here, we are assigning the specific value of formdata into variable

File: app/routes/$workspaceId/settings.tsx [hunk 2]
```diff
@@ -95,7 +95,8 @@ export default function Settings() {
   const { currentWorkspaceId } = useLoaderData()
 
   useEffect(() => {
-    if (location.pathname === "/settings") return navigate("/settings/general")
+    if (location.pathname === "/settings")
+      return navigate("/settings/workspace")
   }, [navigate, location.pathname])
   useEffect(() => {
```
Comment (yash6102000) [inline app/routes/$workspaceId/settings.tsx:101]: add this route into the route constant file

File: app/routes/$workspaceId.tsx (hunk 1 above)
Comment (ayushicopods) [reply app/routes/$workspaceId.tsx:58]: didn't get your point

File: app/routes/$workspaceId/settings.tsx [hunk 3]
```diff
@@ -95,7 +95,8 @@ export default function Settings() {
   const { currentWorkspaceId } = useLoaderData()
 
   useEffect(() => {
-    if (location.pathname === "/settings") return navigate("/settings/general")
+    if (location.pathname === "/settings")
```
Comment (anuragP04) [inline app/routes/$workspaceId/settings.tsx:98]: pick route from routeConstants

File: app/components/settings/SettingTab.tsx [hunk 4]
```diff
@@ -9,10 +9,6 @@ const SettingsTabs = ({
 }) => {
   const { t } = useTranslation()
   const tabs = [
```
Comment (anuragP04) [inline app/components/settings/SettingTab.tsx:11]: don't remove this code, comment it out only

**Review by anuragP04 — CHANGES_REQUESTED** [review]

File: app/components/settings/ResetPassword.tsx [hunk 5]
```diff
@@ -155,55 +166,53 @@ const ResetPassword = ({
...
+            return (
+              <PasswordInputFields
+                onBlur={() => comparePasswords(newPassword, confirmPassword)}
+                {...props}
+                key={props.name}
+              />
+            )
+          })}
+        </div>
+        {newPassword ? (
```
Comment (yash6102000) [inline app/components/settings/ResetPassword.tsx:181]: can we write it as newPassword&&

**Review by Dhruv-Copods — APPROVED** [review]

File: app/components/settings/ResetPassword.tsx [hunk 6]
```diff
@@ -155,55 +166,53 @@ const ResetPassword = ({
...
-          </div>
-          {newPassword ? (
-            <span className="flex gap-1 text-sm">
-              {t("commonConstants.passwordStrength")}:
-              <span className={getPasswordStrengthColor(passwordStrength)}>
-                {passwordStrength}
-              </span>
+      <div className="flex flex-col gap-2">
+        <div className="input-container-wrapper flex flex-col gap-6">
+          {PasswordInputFieldProps.map((props) => {
```
Comment (yash6102000) [inline app/components/settings/ResetPassword.tsx:171]: please rename it as passwordInputFieldProps .....

**Review by yash6102000 — CHANGES_REQUESTED** [review]

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**